doc/examples/perception_pipeline/CMakeLists.txt

```diff
@@ -1,13 +1,17 @@
-add_executable(cylinder_segment src/cylinder_segment.cpp)
-target_link_libraries(cylinder_segment ${catkin_LIBRARIES})
+install(DIRECTORY urdf
+  DESTINATION share/${PROJECT_NAME})
```

Comment:
We try to avoid the ${PROJECT_NAME} variable and use the concrete project name to avoid errors since last year. Do you mind updating that for this tutorial?

Reply:
I changed ${PROJECT_NAME} with moveit2_tutorials. Do you think it's make sense?

Reply:
Thanks! The best practice hasn't made it yet into the tutorials package. But I'd like to start with that here